examples/ensemble/plot_gradient_boosting_quantile.py

```diff
@@ -153,11 +153,11 @@ def highlight_min(x):
 #
 # Note that because the target distribution is asymmetric, the expected
 # conditional mean and conditional median are signficiantly different and
-# therefore one could not use the least squares model get a good estimation of
+# therefore one could not use the squared error model get a good estimation of
 # the conditional median nor the converse.
 #
 # If the target distribution were symmetric and had no outliers (e.g. with a
-# Gaussian noise), then median estimator and the least squares estimator would
+# Gaussian noise), then median estimator and the squared error estimator would
```

Comment:
"squared error estimator" -> 22k occurrences on Google
"least squares estimator" -> 370k occurrences

So I think it's preferable to still talk about the least squared estimator in this example, we just need make the correspondence to the loss name explicit.